data/create-data.sh

```diff
@@ -1,61 +1,62 @@
 echo "= Init create data"
 echo "= Delete caches"
-curl -XDELETE  http://localhost:11222/rest/v2/caches/users
-curl -XDELETE  http://localhost:11222/rest/v2/caches/managers
-curl -XDELETE  http://localhost:11222/rest/v2/caches/programmers
-curl -XDELETE  http://localhost:11222/rest/v2/caches/seniors
-curl -XDELETE  http://localhost:11222/rest/v2/caches/people
-curl -XDELETE  http://localhost:11222/rest/v2/caches/example
-curl -XDELETE  http://localhost:11222/rest/v2/caches/local_users
-curl -XDELETE  http://localhost:11222/rest/v2/caches/invalidated
-curl -XDELETE  http://localhost:11222/rest/v2/caches/invalidated-sync
-curl -XDELETE  http://localhost:11222/rest/v2/caches/xsiteCache
-curl -XDELETE  http://localhost:11222/rest/v2/caches/xsite-transactional
+curl -XDELETE  -u igor:pass http://localhost:11222/rest/v2/caches/users
```

Comment:
set the username and password in variables so they can be easily changed.

Reply:
done